README.md

```diff
@@ -2,391 +1,310 @@
-as well as [OpenJDK](http://openjdk.java.net/) with matching versions.
+* [Overview: what does it do?](#overview-what-does-it-do)
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [Quick start guide](#quick-start-guide)
+* [Next steps: additional documentation](#next-steps-additional-documentation)
+* [Contributing to this project](#contributing-to-this-project)
+* [License](#license)
+* [Usage tracking](#usage-tracking)
+* [Further reading](#further-reading)
 
-| Version			| End of Life   |
-|-------------------------------|:-------------:|
-| Oracle JDK8 / OpenJDK 1.8	| 2019-01	|
-| Oracle JDK11			| TBA		|
+## Overview: what does it do?
+The Learnosity Java SDK makes it simple to interact with Learnosity APIs.
 
-Earlier versions may still keep working. Please refer to our [build
-matrix](https://travis-ci.org/Learnosity/learnosity-sdk-java) to verify (see the
-`Allowed Failures` section). However, please plan to upgrade to a supported JDK
-at the earliest opportunity.
+![image-concept-overview.png](docs/images/image-concept-overview.png)
 
-## Installation
-Installation can be done in three ways:
+It provides a number of convenience features for developers, that make it simple to do the following essential tasks:
+* Creating signed security requests for API initialization, and
+* Interacting with the Data API.
 
-1.) Download the jar in directory learnosity-sdk-java/Dist/All/ and add to your build path. All the required libraries are included in this jar.
+For example, the SDK helps with creating a signed request for Learnosity:
 
-2.) Download the jar in learnosity-sdk-java/Dist/Light/ and add to your build path. This requires the external libraries which can be found in learnosity-sdk-java/Dist/Light/RequiredJars
+![image-signed-request-creation.png](docs/images/image-signed-request-creation.png)
 
-3.) Download the source code and integrate into your project. This requires the external libraries which can be found in learnosity-sdk-java/Dist/Light/RequiredJars
+Once the SDK has created the signed request for you, your app sends that on to an API in the Learnosity cloud, which then retrieves the assessment you are asking for, as seen in the diagram below:
 
-4.) To use as a dependency in a maven project see below 'Using with a file based repository'
+![image-assessment-retrieval.png](docs/images/image-assessment-retrieval.png)
 
-## Test
+This scenario is what you can see running in the Quick start guide example ([see below](#quick-start-guide)).
 
-In order to check that you've added the required code correctly, you can download src/LearnositySdk/Test and try to run the test file. If it runs successfully, all should be fine.
+There's more features, besides. See the detailed list of SDK features on the [reference page](REFERENCE.md).
 
-The Test.java class also gives some examples on how to use the sdk.
+[(Back to top)](#table-of-contents)
 
-## Usage
+## Requirements
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 0; Below](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
## Requirements

```